monai/apps/auto3dseg/bundle_gen.py

```diff
@@ -511,15 +520,20 @@ def generate(
                 gen_algo.set_data_stats(data_stats)
                 gen_algo.set_data_source(data_src_cfg)
                 name = f"{gen_algo.name}_{f_id}"
-                if gpu_customization:
-                    gen_algo.export_to_disk(
-                        output_folder,
-                        name,
-                        fold=f_id,
-                        gpu_customization=True,
-                        gpu_customization_specs=gpu_customization_specs,
-                    )
+                gen_algo.pre_check_skip_algo()
+                if not gen_algo.skip_bundlegen:
+                    if gpu_customization:
+                        gen_algo.export_to_disk(
+                            output_folder,
+                            name,
+                            fold=f_id,
+                            gpu_customization=True,
+                            gpu_customization_specs=gpu_customization_specs,
+                        )
+                    else:
+                        gen_algo.export_to_disk(output_folder, name, fold=f_id)
+
+                    algo_to_pickle(gen_algo, template_path=algo.template_path)
+                    self.history.append({name: gen_algo})  # track the previous, may create a persistent history
                 else:
-                    gen_algo.export_to_disk(output_folder, name, fold=f_id)
-                algo_to_pickle(gen_algo, template_path=algo.template_path)
-                self.history.append({name: gen_algo})  # track the previous, may create a persistent history
+                    logger.info(f"{name} is skipped! Msg:{gen_algo.skip_info}")
```

Comment:
The Msg is empty by default and the message `Msg: ` will look confusing.

Reply:
removed msg: